### PR TITLE
OHRM5X-270 : Test scripts for job employment status screen

### PR DESCRIPTION
**Pull request by ShenaliFernando**

### CODE REVIEW

File: symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import user from '/home/administrator/Desktop/OS /orangehrm/symfony/test/functional/cypress/fixtures/user.json'
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js:1]: don't commit pull-path

File: symfony/test/functional/cypress/fixtures/user.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
     "admin": {
         "userName": "Admin",
-        "password": "Admin@123",
-        "fullName": "Samantha Jayasinghe"
+        "password": "admin@OHRM123",
+        "fullName": "Jacqueline White"
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/fixtures/user.json:5]: {your-admin-pwd} goes here

File: symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    })
+
+//Add a new employment status   
+ describe ('add new employment status', function(){
+
+        it ('check add new employment status', () => {
+    
+            cy.login (user.admin.userName, user.admin.password) 
+            cy.visit ('/admin/saveEmploymentStatus')
+            cy.xpath('//*[@id="app"]/div[1]/div/div[2]/div/div/form/div[1]/div/div[2]/input').type('Full time')
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js:31]: xpath may need to change

File: symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+        it('validate add employment status by exceeding the character no', () => {
+    
+            cy.login(user.admin.userName, user.admin.password)
+            cy.visit( '/admin/saveEmploymentStatus')
+            cy.xpath('//*[@id="app"]/div[1]/div/div[2]/div/div/form/div[1]/div/div[2]/input').type('validate add employment status field by exceeding the character no')
+            cy.get ('form').submit()
+            cy.get('.oxd-input-group__message').should('include.text', 'Should not exceed 50 characters')
+    
+
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js:47]: extra spaces

File: symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    
+        it('validate add employment status required field', () => {
+    
+            cy.login(user.admin.userName, user.admin.password) 
+            cy.visit( '/admin/saveEmploymentStatus')  
+            cy.xpath('//*[@id="app"]/div[1]/div/div[2]/div/div/form/div[1]/div/div[2]/input').type(' ')   
+            cy.get ('form').submit()
+            cy.get('.oxd-input-group__message').should('include.text', 'Required')
+    
+
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js:62]: extra spaces

File: symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js (hunk 1 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js]: Pull path removed

File: symfony/test/functional/cypress/fixtures/user.json (hunk 2 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/fixtures/user.json:5]: Created a separate file as admin.json and added admin credentials inside it.

File: symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js (hunk 3 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js]: xpath removed from the code.

File: symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js (hunk 4 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/integration/orangehrm/admin/jobemploymentstatus.spec.js]: Extra space removed

**Review by samanthajayasinghe — APPROVED** [review]

**Pull request merged**